## HYPOTHESIS TESTING

### Background:
#### Bombay hospitality Ltd. operates a franchise model for producing exotic Norwegian dinners throughout New England. The operating cost for a franchise in a week (W) is given by the equation W = $1,000 + $5X, where X represents the number of units produced in a week. Recent feedback from restaurant owners suggests that this cost model may no longer be accurate, as their observed weekly operating costs are higher.
### Objective:
#### To investigate the restaurant owners' claim about the increase in weekly operating costs using hypothesis testing.
### Data Provided:
#### • The theoretical weekly operating cost model: W = $1,000 + $5X
#### • Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
#### • Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


### Assignment Tasks:
## 1. State the Hypotheses statement:


#### A hypothesis statement is a clear and testable prediction about the relationship between variables in a study. To formulate a hypothesis, consider the research question or problem statement.

## 2. Calculate the Test Statistic:
### Use the following formula to calculate the test statistic (t):
### where:
### •	ˉxˉ = sample mean weekly cost (Rs. 3,050)
### •	μ = theoretical mean weekly cost according to the cost model (W = $1,000 + $5X for X = 600 units)
### •	σ = 5*25 units
### •	n = sample size (25 restaurants)


In [2]:
import math

x_bar = 3050  
mu = 4000  
sigma = 5 * 25 
n = 25 

SE = sigma / math.sqrt(n)

t_statistic = (x_bar - mu) / SE
t_statistic


-38.0

## 3. Determine the Critical Value:
### Using the alpha level of 5% (α = 0.05), determine the critical value from the standard normal (Z) distribution table.


In [3]:
import scipy.stats as stats

df = n - 1

alpha = 0.05
t_critical = stats.t.ppf(1 - alpha, df)
t_critical


1.7108820799094275

#### The critical t-value for a one-tailed test at a 5% significance level (α = 0.05) with 24 degrees of freedom is 1.71.

#### Since the computed t-statistic (-38.0) is far less than the critical value (1.71), we fail to reject the null hypothesis. This means there is no statistical evidence to support the claim that the weekly operating costs have increased.

## 4. Make a Decision:
### Compare the test statistic with the critical value to decide whether to reject the null hypothesis.


### •The Calculated t-statistic = -38.0
### •The critical value for α = 0.05(one tailed test)=1.711
### •Decision Rule:
    ### •If t ≥ 1.711, reject 𝐻0.
    ### •If t < 1.711, fail to reject 𝐻0.
    

In [6]:
if t_statistic > t_critical:
    decision = "Reject the null hypothesis: There is significant evidence that the weekly operating costs have increased."
else:
    decision = "Fail to reject the null hypothesis: There is no significant evidence that the weekly operating costs have increased."

decision


'Fail to reject the null hypothesis: There is no significant evidence that the weekly operating costs have increased.'

#### Since the t-statistic (-38.0) is much lower than the critical t-value (1.71), we fail to reject the null hypothesis.

#### This means there is no significant evidence to support the claim that the weekly operating costs have increased.

## 5. Conclusion:
### Based on the decision in step 4, conclude whether there is strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests.


#### Based on the hypothesis test, we fail to reject the null hypothesis. The test statistic 𝑡=−38.0 is significantly lower than the critical value 𝑡=1.711 indicating that the observed weekly operating costs   3,050 are actually much lower than the predicted cost  4,000 according to the given cost model.

### Implication:
#### There is no strong statistical evidence to support the restaurant owners' claim that weekly operating costs have increased. Instead, the data suggests that the actual costs are significantly lower than what the model predicts. This contradicts the concerns of the restaurant owners and suggests that further investigation may be needed to understand why their perceptions differ from the data.